sklearn/metrics/scorer.py

```diff
@@ -201,8 +201,7 @@ def _passthrough_scorer(estimator, *args, **kwargs):
     return estimator.score(*args, **kwargs)
 
 
-def check_scoring(estimator, scoring=None, allow_none=False,
```

Comment:
I guess it comes from 5010f393a24aa380bf33bee1a7f7a107ad6da084 and can savely be removed. The thing is: if this is considered public API, you can not just remove a parameter, it needs a deprecation cycle. I am not sure this should be considered public API, though.
A bit of an edge-case. Maybe it is fine.